ide/static/ide/js/settings.js

```diff
@@ -240,6 +240,11 @@ CloudPebble.Settings = (function() {
             }
         });
 
+        pane.find('#uuid-generate').click(function() {
+            var uuid_field = settings_template.find('#settings-uuid');
+            uuid_field.val(_.UUID.v4)
```

Comment:
While jQuery does permit passing a function, which it will then call itself, I'd prefer to explicitly invoke this and return the result.